src/mgr/DaemonServer.h

```diff
@@ -272,7 +272,7 @@ class DaemonServer : public Dispatcher, public md_config_obs_t
   ~DaemonServer() override;
 
   bool ms_dispatch2(const ceph::ref_t<Message>& m) override;
-  int ms_handle_fast_authentication(Connection *con) override;
+  bool ms_handle_fast_authentication(Connection *con) override;
```

Comment:
Proposal: how about marking this as `[[nodiscard]]` to rise a warning if the return goes unchecked?

Reply:
Sure!